### PR TITLE
Updated docs according to the issue #759

### PR DESCRIPTION
**Pull request by LeftTry**
According to the [rspamd/rspamd.com#759](https://github.com/rspamd/rspamd.com/issues/759) documentation for `replies` and `known_senders` modules were updated. A template for release notes was also provided.

### CODE REVIEW

File: doc/modules/known_senders.md [hunk 1]
```diff
@@ -71,3 +82,16 @@ symbol_unknown = 'UNKNOWN_SENDER';
...
 
 - **Description**: The `symbol_unknown` parameter specifies the symbol that will be inserted for new senders from the domains listed in the `domains` configuration. This symbol can be used to further classify emails from unknown senders.
+
+### Verified Incoming Mail Global Symbol
+
+- **Description**: The `symbol_check_mail_global` parameter specifies the symbol that will be inserted if sender in the incoming mail is verified by `replies` module.
+ 
+### Verified Incoming Mail Local Symbol
+
+- **Description**: The `symbol_check_mail_global` parameter specifies the symbol that will be inserted if recipients in the incoming mail is verified by `replies` module.
```
Comment (moisseev) [inline doc/modules/known_senders.md:92]: It's supposed to be `symbol_check_mail_local`.

File: _posts/2024-02-26-rspamd-3.8.template.md
Comment (moisseev) [inline _posts/2024-02-26-rspamd-3.8.template.md:1]: I believe adding the `template` suffix won't prevent this post from publishing. Probably you should create the `_drafts` folder and put it in there instead.

Comment (vstakhov) [reply _posts/2024-02-26-rspamd-3.8.template.md:1]: You can also place that content just in the PR comment, so I will be able to use it when drafting release notes.

**Comment by LeftTry** [comment]
So as @vstakhov suggested I will leave paragraph for release notes here and also duplicated it to the _darfts folder as @moisseev suggested:


This release includes new features for `replies` and `known_senders` modules. `replies` module now can check `in-reply-to` mails to verify senders and their recipients. `known_senders` module can check incoming mails if sender and recipients have already been verified.

**Review by vstakhov — APPROVED** [review]

**Pull request merged**